packages/jest-runner/src/runTest.ts

```diff
@@ -106,13 +106,17 @@ async function runTestInternal(
   const TestEnvironment: typeof JestEnvironment = interopRequireDefault(
     require(testEnvironment),
   ).default;
-  const testFramework: TestFramework =
+  const testFramework: TestFramework = interopRequireDefault(
```

Comment:
can use `import()` here with built-in interop, but that creates extra promises 🤷 Also not sure if that works with real ESM